src/Algebra/Properties/BooleanAlgebra.agda

```diff
@@ -26,8 +26,7 @@ open BooleanAlgebra B
 
 import Algebra.Properties.DistributiveLattice as DistribLatticeProperties
 
-open import Algebra.Structures _≈_
```

Comment:
Should probably be placed lower down in the ordering? Or is it needed for subsequent `import`s?

Reply:
My mistake: this is another deprecated module, so there really shouldn't be any commits to it, unless/until it gives rise to errors during CI?

Reply:
Right, as @jamesmckinna says, don't change any deprecated module, it is not worth it. These will eventually be deleted.